pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -472,6 +515,26 @@ func fixCrcDnsmasqConfigFile() error {
 	return nil
 }
 
+func removeCrcDnsmasqConfigFile() error {
+	if _, err := os.Stat(crcDnsmasqConfigPath); !os.IsNotExist(err) {
```

Comment:
Getting back to the discussion from above regarding handling of `skip-`, The error is ignored in case the user disabled dnsmasq?

Reply:
@cfergeau yes we first check if file is exist before doing any other operation to it.

Reply:
But why? On a system where `crc setup` successfully ran, this file will be present, so why do we test if it's there, and ignore it if missing?

Reply:
@cfergeau what is someone use `skip-check-crc-dnsmasq-file` during the setup as second run then this file will be there and `crc setup` will also successfully running, this is just ensuring remove of the file which we might create.

Reply:
Ok, so this goes back to a discussion I started somewhere else during this review. This is how you want to handle skipped checks. Which is probably fine, but needs to be explained.

Reply:
@cfergeau where you want it to be explained in commit log or as part of comment?

Reply:
it probably fits better in the commit log as this is a general description of how things are expected to work, and why the code is written this way.

Reply:
Done.